Countdown-timer/css/style.css

```diff
@@ -0,0 +1,42 @@
+main {
+  background-color: blanchedalmond;
+  height: 200px;
+  width: 60%;
+  margin: 0 auto;
+  padding: 20px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  resize: both;
+  overflow: auto;
+}
+#main_container{
+    display: flex;
+}
+
+input{
+    border-radius: 5px;
+    width: 40px;
+    border: 1px solid black;
+    margin-right: 3px;
+    padding:10px
+    
+}
+button{
+    background-color: cornflowerblue;
+    border-radius: 6px;
+    padding: 10px;
+    margin-left: 10px;
```

Comment:
Never play with `margin`. Always use `padding` for such cases. Padding doesn't break css with other containers.